CHANGELOG-1.2.md

```diff
@@ -0,0 +1,92 @@
+# Release notes for 1.2.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.1.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Fix a bug that when CSI migration is enabled and PV is using GA topology label, resizing is not work. ([#139](https://github.com/kubernetes-csi/external-resizer/pull/139), [@Jiawei0227](https://github.com/Jiawei0227))
```

Comment:
should this be kind/bug? or kind/feature "add support for csi migration for GA topology labels"

Reply:
added under bug.